docs/admin/auth/methods.rst

```diff
@@ -137,6 +137,11 @@ Token must contain the following claims:
 ``CREATE USER`` statement, the cluster id is used and must match the token's
 ``aud`` value. See :ref:`create-user-jwt` for details.
 
+For users, created without JWT properties,
+ref:`default JWT values <external_auth>` are used.
+``iss`` and ``aud`` must match default values.
+``username`` field defines CrateDB user.
+
```

Comment:
I think the above text should also be modified slightly to account for the changes.
For example:


```diff
diff --git a/docs/admin/auth/methods.rst b/docs/admin/auth/methods.rst
index 230ddb639cf..d1640225898 100644
--- a/docs/admin/auth/methods.rst
+++ b/docs/admin/auth/methods.rst
@@ -118,11 +118,11 @@ external service.
 
 The external service is responsible for issuing a `JWT`_ access token for the
 user. The user then provides this token, prefixed with ``Bearer`` in the
 ``Authorization`` HTTP header to CrateDB.
 CrateDB will validate the token and match it to a user created with ``CREATE
-USER`` with ``JWT`` properties that match those of the provided ``JWT`` token.
+USER``.
 
 Token must contain the following claims:
 
 ``kid`` - `Key ID`_.
 
@@ -130,14 +130,19 @@ Token must contain the following claims:
 
 ``aud`` - `aud`_.
 
 ``username`` - user name in a third party app.
 
-``iss``, ``username`` and ``aud`` values must match the values created by
-``CREATE USER`` statement. If ``aud`` has not been defined on the
-``CREATE USER`` statement, the cluster id is used and must match the token's
-``aud`` value. See :ref:`create-user-jwt` for details.
+The token's ``iss``, ``username`` and ``aud`` values must match the
+corresponding values within CrateDB.
+CrateDB uses either the values defined on a user level via ``CREATE USER`` or if
+absent the global configuration.
+
+If ``aud`` defaults to the cluster id if absent from both the user definition
+and the global configuration.
+
+See :ref:`create-user-jwt` and LINK_TO_GLOBAL_CONFIG for details.
 
 It's recommended to have ``exp`` (`expiration date`_ as epoch seconds) in the
 header. If it's provided, the token's expiration date will be checked against
 the local system's time in UTC.
 ```

Reply:
I applied it in https://github.com/crate/crate/pull/16763/commits/ecac57c5cf6e8ec84c3da68ffeda38c432e2de9d but removed original text as it's basically repetition of the same thing. 

So current version already covers everything in a single sentence.